sbt_common/internal_model/src/test/scala/uk/ac/wellcome/models/work/test/util/IdentifiersGenerators.scala

```diff
@@ -22,14 +22,63 @@ trait IdentifiersGenerators {
       ontologyType = ontologyType
     )
 
-  def createSierraSourceIdentifier = createSierraSourceIdentifierWith()
+  def createSierraSourceIdentifier: SourceIdentifier =
+    createSierraSourceIdentifierWith()
 
   def createSierraSourceIdentifierWith(
     value: String = randomAlphanumeric(length = 10),
     ontologyType: String = "Work"): SourceIdentifier =
     SourceIdentifier(
-      identifierType = IdentifierType("sierra-system-number"),
+      identifierType = createSierraSourceIdentifierType,
       value = value,
       ontologyType = ontologyType
     )
+
+  def createIsbnSourceIdentifier: SourceIdentifier =
+    createIsbnSourceIdentifierWith()
+
+  def createIsbnSourceIdentifierWith(
+    value: String = randomAlphanumeric(length = 10),
+    ontologyType: String = "Work"): SourceIdentifier =
+    SourceIdentifier(
+      identifierType = createIsbnSourceIdentifierType,
+      value = value,
+      ontologyType = ontologyType
+    )
+
+  def createMiroSourceIdentifier: SourceIdentifier =
+    createMiroSourceIdentifierWith()
+
+  def createMiroSourceIdentifierWith(
```

Comment:
minor: this method is only ever called in `createMiroSourceIdentifier`, so I’d inline it there and do away with this method.

My pattern when writing these generators elsewhere has been to write as little as possible – and if I don’t need a `With()` method with arguments, we don't have it.

Reply:
Ditto for createIsbnSourceIdentifierWith and createMiroLibraryReferenceIdentifierWith.